scheduler/src/cook/rest/api.clj

```diff
@@ -1062,9 +1062,8 @@
   [instance-entity]
   (if-let [sandbox-url (:instance/sandbox-url instance-entity)]
     sandbox-url
-    (if-let [compute-cluster (task/task-ent->ComputeCluster instance-entity)]
-      (cc/retrieve-sandbox-url-path compute-cluster instance-entity)
-      (log/error "Unable to get sandbox URL for" instance-entity "whose compute cluster resolves to nil"))))
```

Comment:
Why are we getting rid of this error log?

Reply:
this is no longer an error. We don't expect to find dynamic clusters on a Cook follower. The logic above, [sandbox-url (:instance/sandbox-url instance-entity)], already does what a k8s cluster would do.